numpy/random/tests/test_generator_mt19937.py

```diff
@@ -1069,23 +1069,25 @@ def test_dirichlet_small_alpha(self):
         random = Generator(MT19937(self.seed))
         actual = random.dirichlet(alpha, size=(3, 2))
         expected = np.array([
-            [
-                [1., 0.],
-                [1., 0.]
-            ],
-
-            [
-                [1., 0.],
-                [1., 0.]
-            ],
-
-            [
-                [1., 0.],
-                [1., 0.]
-            ]
+            [[1., 0.],
+            [1., 0.]],
```

Comment:
Sorry to nitpick, but the opening bracket needs one more space in front of it, so it aligns with the other sublist above it. (Likewise for the similar cases below.)

Reply:
Thanks for being so thorough. This is fixed now as suggested.